benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/mysql/MysqlDataLoader.java

```diff
@@ -31,6 +51,64 @@
    */
   @Override
   public void loadData(DataFormat data) throws Exception {
+    if (!(data instanceof MysqlDataFormat)) {
+      throw new IllegalArgumentException("Wrong data format for MySQL.");
+    }
+
+    Class.forName("com.mysql.cj.jdbc.Driver");
+    connection = DriverManager.getConnection(url);
+    statement = connection.createStatement();
+
+    createDatabase();
+    createTables();
+
+    Map<String, LinkedList<String>> list = ((MysqlDataFormat) data).getTableDataFilesList();
+
+    for (String table : list.keySet()) {
+      for (String file : list.get(table)) {
+        String query =
+            "LOAD DATA LOCAL INFILE '" + file + "' INTO TABLE " + MysqlTpchSchema.databaseName + "."
+                + table + " FIELDS TERMINATED BY ',' ENCLOSED BY '\"'"
+                + " LINES TERMINATED BY '\n'";
+        statement.executeUpdate(query);
+      }
+    }
+  }
+
+  private void createDatabase() throws Exception {
+    statement.executeUpdate("SET GLOBAL local_infile=1;");
+    statement.executeUpdate("create database " + MysqlTpchSchema.databaseName);
+  }
+
+  private void createTables() throws Exception {
+    for (String tablename : MysqlTpchSchema.schemaMap.keySet()) {
+      String sql = "create table " + MysqlTpchSchema.databaseName + "." + tablename + " (";
+      for (String field : MysqlTpchSchema.schemaMap.get(tablename).keySet()) {
+        sql += field + " " + MysqlTpchSchema.schemaMap.get(tablename).get(field) + ", ";
+      }
+
+      sql += "primary key ( ";
+      int index = 1;
+      for (String key : MysqlTpchSchema.primaryKeyMap.get(tablename)) {
+        sql += key;
+        if (index++ < MysqlTpchSchema.primaryKeyMap.get(tablename).size()) {
+          sql += ", ";
+        }
+      }
+      sql += ")";
+
+      if (MysqlTpchSchema.foreignKeyMap.containsKey(tablename)) {
+        for (String field : MysqlTpchSchema.foreignKeyMap.get(tablename).keySet()) {
+          sql += ", foreign key (" + field + ") references ";
+          for (String key : MysqlTpchSchema.foreignKeyMap.get(tablename).get(field).keySet()) {
```

Comment:
StringJoiner...

Reply:
this part is adding foreign key syntax. it will have only 2 values so didn't use here.

Reply:
We should still use it, as you are thinking about the current case it's used in, not potential future cases.

Reply:
added this change.